sci-physics/geant4_vmc/files/geant4_vmc-6.0-cmake-g4root-tests.patch

```diff
@@ -0,0 +1,28 @@
+From 5d25b7fd6d53b620b25bed3a72d739d1644b5731 Mon Sep 17 00:00:00 2001
```

Comment:
I'd include links to relevant Gentoo bug(s) at the top of patches, as well as upstream PRs/commits, but I know that you've given it in the ebuild for the PR link anyway.

Reply:
That's a great model, and easier to consume than comments for sure. 
I force-pushed just now adding:
```
Pull Request: https://github.com/vmc-project/geant4_vmc/pull/28
Bug: https://bugs.gentoo.org/833126
```
to the `.patch`. Thanks!